app/models/settings/user_experience.rb

```diff
@@ -21,7 +21,7 @@ class UserExperience < Base
     }
     # a non-public forem will redirect all unauthenticated pages to the registration page.
     # a public forem could have more fine-grained authentication (listings ar private etc.) in future
-    setting :public, type: :boolean, default: 0
+    setting :public, type: :boolean, default: true
```

Comment:
Since the type here is boolean it makes set the value to a boolean too. In addition,  we've decided to set public to `true` based on [this](https://forem-team.slack.com/archives/C01S4QZUDNC/p1638896683273800) conversation